tests/integration/molecule/module_event/playbook.yml

```diff
@@ -18,6 +18,187 @@
           - result is failed
           - "result.msg == \"missing parameter(s) required by 'check': entity\""
 
+    - name: Create event with missing objects on remote
+      event:
+        auth:
+          url: http://localhost:8080
+        timestamp: 134532453
+        entity: entity
+        check: check
+      ignore_errors: true
+      register: result
+
+    - assert:
+        that:
+          - result is failed
+          - result.msg == "Entity with name 'entity' does not exist on remote."
+
+    - name: Create simple entity
+      entity:
+        auth:
+          url: http://localhost:8080
+        name: awesome_entity
+        entity_class: proxy
+
+    - name: Create event with missing check on remote
+      event:
+        auth:
+          url: http://localhost:8080
+        timestamp: 134532453
+        entity: awesome_entity
+        check: check
+      ignore_errors: true
+      register: result
+
+    - assert:
+        that:
+          - result is failed
+          - result.msg == "Check with name 'check' does not exist on remote."
+
+    - name: Create a simple check
+      check:
+        auth:
+          url: http://localhost:8080
+        name: awesome_check
+        command: /bin/true
+        subscriptions:
+          - checks
+          - also_checks
+        interval: 30
+
+    - name: Create event with minimal parameters
+      event:
+        auth:
+          url: http://localhost:8080
+        entity: awesome_entity
+        check: awesome_check
+      register: result
+
+    - assert:
+        that:
+          - result is changed
+          - result.object.check.metadata.name == 'awesome_check'
+          - result.object.check.command == '/bin/true'
+          - result.object.entity.metadata.name == 'awesome_entity'
+
+    - name: Get last event
+      event_info:
+        auth:
+          url: http://localhost:8080
+        entity: awesome_entity
+        check: awesome_check
+      register: result
+
+    - assert:
+        that:
+          - result.objects.0.check.metadata.name == 'awesome_check'
+          - result.objects.0.check.command == '/bin/true'
+          - result.objects.0.entity.metadata.name == 'awesome_entity'
+
+    - name: Get all events
+      event_info:
+        auth:
+          url: http://localhost:8080
+      register: result
+
+    - assert:
+        that:
+          - result.objects | length == 1
+
+    - name: Create second entity
+      entity:
+        auth:
+          url: http://localhost:8080
+        name: entity2
+        entity_class: proxy
+      register: entity
```

Comment:
I could be wrong, but I think we're not using the returned `entity` anywhere, so we can we can remove this?